cypress/e2e/spec-html.cy.js

```diff
@@ -57,6 +57,17 @@ it("blocks external links", () => {
     .should("include.text", "An error has occurred");
 });
 
+it("allows permitted external links", () => {
+  localStorage.clear();
+  cy.visit(baseUrl);
+  cy.get("div[class=cm-content]").invoke(
+    "text",
+    '<a href="https://www.rpf.io/seefood">some external link</a>',
+  );
+  cy.get(".btn--run").click();
+  cy.url().should("not.equal", baseUrl);
+});
+
```

Comment:
I think I don't quite understand this test... Because we don't seem to be simulating the click on the actual link (that I can see). Although I'm not sure how easy it would be to test the content of the external link page, especially if we just put in the preview iframe as Marc was suggesting 🤔  Unless what we want to do is run the code, find the link in the iframe and then check that the href has not been replaced...

Reply:
Yes you're right about this! My error